frontend-site/src/modules/staticData.js

```diff
@@ -53,6 +54,12 @@ export const servicesOffered = Object.freeze([
     description: 'Need a break a stressful week of classes? Pick up a board game, located in the LUG office, and grab some friends. There are plenty of games to choose from.',
     url: '',
   },
+  {
+    name: 'Minecraft',
+    image: require("@/assets/minecraft.jpg"),
+    description: 'If you still play this iconic game, come join the LUG Minecraft server. To learn more about joing the server, please select the  button below to be connected to our #game slack channel.',
```

Comment:
(Minor) `#game` -> `#games` and `slack` -> `Slack` and `joing` -> `joining`. Also, you have 2 spaces in `please select the  button`